ecs-cli/modules/commands/flags/flags.go

```diff
@@ -191,3 +192,19 @@ func UsageErrorFactory(command string) func(*cli.Context, error, bool) error {
 		return err
 	}
 }
+
+func CFNResourceFlags() []string {
```

Comment:
Should add `--capability-iam` to this list

Reply:
This is what happens when it's included:
```
~/go/src/github.com/aws/amazon-ecs-cli(empty-cluster)$ ecs-cli up -e --cluster e3
WARN[0000] Value for flag 'capability-iam' will be ignored when creating an empty cluster 
```

While having the flag in the list seems logical, this output is confusing since the flag was not in fact included.